styles/_article.scss

```diff
@@ -86,12 +86,39 @@
     > p:last-child { margin-bottom: 0; }
   }
 
+  &-anchorbox {
+    background-color: #e8e8e8;
+    padding: 15px;
+    font-weight: $font-weight-bold;
+    margin-bottom: 30px;
+
+    & > p {
+      margin-top: 0px;
+      margin-bottom: 10px;
+    }
+
+    & > ul {
+      margin-bottom: 0px;
+      margin-top: 0px;
+      margin-left: 5px;
```

Comment:
I'm okay adding this for now, but I think we'll want to revisit

Reply:
let me know if you want me to change anything here